ui/ui-components/pages/app/[id]/refresh.tsx

```diff
@@ -61,10 +63,10 @@ export default function Page(props) {
 
     if (response?.appRefreshQuery) {
       setAppRefreshQuery(response.appRefreshQuery);
```

Comment:
@evantahler RE: `locked=undefined`...it doesn't even get as far as the place you mentioned (in the `edit` function).  If I `console.log()` the response and what it sets the `appRefreshQuery` variable as, they both also have `locked = undefined`

Reply:
Can you check in the `getInitialProps` - to make sure we are starting in a good place?

Reply:
yep it's reading as `null` there

Reply:
I paired with @krishnaglick  on it and couldn't get it to fail for some reason.  We did update `disabled` to be a `useMemo`.  Seems to be working great.